ports/stm32/omv_spi.c

```diff
@@ -330,8 +330,8 @@ static int omv_spi_dma_init(omv_spi_t *spi, uint32_t direction, omv_spi_config_t
     dma_init->Direction = direction;
     dma_init->SrcInc = (direction == DMA_PERIPH_TO_MEMORY) ? DMA_SINC_FIXED : DMA_SINC_INCREMENTED;
     dma_init->DestInc = (direction == DMA_MEMORY_TO_PERIPH) ? DMA_DINC_FIXED : DMA_DINC_INCREMENTED;
-    dma_init->SrcDataWidth = (config->datasize == 8) ?  DMA_SRC_DATAWIDTH_BYTE : DMA_SRC_DATAWIDTH_WORD;
-    dma_init->DestDataWidth = (config->datasize == 8) ?  DMA_DEST_DATAWIDTH_BYTE : DMA_DEST_DATAWIDTH_WORD;
+    dma_init->SrcDataWidth = DMA_SRC_DATAWIDTH_WORD;
+    dma_init->DestDataWidth = DMA_DEST_DATAWIDTH_WORD;
```

Comment:
Should we do this? What if we're transferring an uneven amount of bytes or shorts?